services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -244,22 +245,23 @@ const deleteIndexedBlocks = async job => {
 		await BluebirdPromise.map(
 			blocks,
 			async block => {
-				let forkedTransactions;
+				let { data: forkedTransactions } = await getTransactionsByBlockID(block.id);
```

Comment:
```suggestion
				const { data: forkedTransactions } = await getTransactionsByBlockID(block.id);
```

Reply:
It cannot be const since we are filtering null transactions here:
https://github.com/LiskHQ/lisk-service/blob/1678-consider-transaction-execution-status-within-the-hooks/services/blockchain-indexer/shared/indexer/blockchainIndex.js#L278